### PR TITLE
Add support for clientUnblock command

### PR DESCRIPTION
**Pull request by dengliming**
Resolves [https://github.com/redis/jedis/issues/2294](https://github.com/redis/jedis/issues/2294)

### CODE REVIEW

File: src/main/java/redis/clients/jedis/UnblockType.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package redis.clients.jedis;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+/**
+ * Unblock type for {@code CLIENT UNBLOCK} command.
+ */
+public enum UnblockType {
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/UnblockType.java:8]: Put this in args package and implement Rawable interface.

File: src/main/java/redis/clients/jedis/UnblockType.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  TIMEOUT, ERROR;
+
+  public final byte[] raw;
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/UnblockType.java:11]:
```suggestion
  private final byte[] raw;
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]

**Comment by dengliming** [comment]
@sazzad16 The package name `args` is very similar to `params`

File: src/main/java/redis/clients/jedis/args/UnblockType.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * Unblock type for {@code CLIENT UNBLOCK} command.
+ */
+public enum UnblockType implements Rawable {
+  TIMEOUT, ERROR;
+
+  private final byte[] raw;
+
+  UnblockType() {
+    raw = SafeEncoder.encode(this.name().toLowerCase());
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/args/UnblockType.java:14]: Any reason to make it lower?
```suggestion
    raw = SafeEncoder.encode(this.name());
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/args/UnblockType.java (hunk 3 above)
Comment (dengliming) [reply src/main/java/redis/clients/jedis/args/UnblockType.java]: No, Let me remove it.

**Comment by sazzad16** [comment]
> The package name `args` is very similar to `params` 

@dengliming What can you suggest us do now? `params` should stay dedicated for _options_. And it is not something that I to break things.

**Review by sazzad16 — COMMENTED** [review]
One last thing. WDYT about using `@Nullable` before `UnblockType`?

**Comment by dengliming** [comment]
> One last thing. WDYT about using @Nullable before UnblockType?

Introduce `@Nullable` maybe better. Let user clearly know the params can be null. but we also need to import the annotation api(e.g. `javax.annotation-api`) right?

**Comment by dengliming** [comment]
> @dengliming What can you suggest us do now? params should stay dedicated for options. And it is not something that I to break things.

Not sure. I can't found a better way to do now. : )

**Comment by sazzad16** [comment]
> Introduce `@Nullable` maybe better. Let user clearly know the params can be null. but we also need to import the annotation api(e.g. `javax.annotation-api`) right? 

Hmm, you're right. Could you please add javadocs (in all methods) stating that `UnblockType` can be `null`? It's either this or adding new methods without `UnblockType`.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**